### PR TITLE
Feat/ssl enforcement

### PR DESCRIPTION
**Pull request by joshenlim**
Normal state:
Screenshot 2023-02-08 at 5 22 54 PM

Error state:
Screenshot 2023-02-08 at 5 28 45 PM

### CODE REVIEW

File: studio/components/interfaces/Settings/Database/SSLConfiguration.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      </div>
+      <FormPanel>
+        <FormSection
+          header={
+            <FormSectionLabel
+              className="lg:col-span-7"
+              description={
+                <div className="space-y-4">
+                  <p className="text-sm text-scale-1000">
+                    Reject connections made to your database without a valid certificate
```
Comment (darora) [inline studio/components/interfaces/Settings/Database/SSLConfiguration.tsx:89]: `without a valid certificate` makes it sound like we're enforcing client cert auth, which we're not. Would suggest "Rejects non-SSL connections to your database" or "Rejects unsecure connections to your database" to keep it simple.

File: studio/data/ssl-enforcement/ssl-enforcement-update-mutation.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  'mutationFn'
+> = {}) => {
+  const queryClient = useQueryClient()
+
+  return useMutation<SSLEnforcementUpdateData, unknown, SSLEnforcementUpdateVariables>(
+    (vars) => updateSSLEnforcement(vars),
+    {
+      async onSuccess(data, variables, context) {
+        const { projectRef } = variables
+        await queryClient.invalidateQueries(sslEnforcementKeys.list(projectRef)),
```
Comment (alaister) [inline studio/data/ssl-enforcement/ssl-enforcement-update-mutation.ts:47]: Cheeky trailing comma here

**Review by alaister — CHANGES_REQUESTED** [review]
Looking good! Appreciate the cleanup too. Just one tiny thing:

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**